config/application.rb

```diff
@@ -37,6 +37,6 @@ class Application < Rails::Application
 
     config.sentry_dsn = ENV["SENTRY_DSN"]&.strip
     config.check_financial_eligibility_host = ENV.fetch("CFE_HOST",
-                                                        "https://check-financial-eligibility-partner-staging.cloud-platform.service.justice.gov.uk/")
+                                                        "http://check-financial-eligibility-partner-check-financial-partner.check-financial-eligibility-partner-staging.svc.cluster.local")
```

Comment:
I don't think this is correct. I believe that this is meant to be a dev/test fallback URL, which is going to fail all the VCR tests. Possibly it needs either documenting or moving into environment-specific settings (one for dev/test/prod)

Reply:
The VCR should be using the value in the local .env file for the CFE host.

As such I have set this to nil and added the URL as an ENV_VAR on circleci. This solves the issue with the deployments, specs and inconsistent CFE_HOST values.

Reply:
I think that can work. If we go down that route, can we add a note to the readme saying that for the tests to pass we'll need a `.env.test` file in root dir and what value should be in it? (Maybe we could update .env.sample with values that will cause the tests to pass?)

Also note that I'm saying `.env.test` - we probably will need the habit of using `.env.test` and `.env.development` instead of `.env.local` if we want to avoid environments breaking one another.